docs/calculations.mdx

```diff
@@ -345,10 +345,10 @@ default_bitrate =
       is_primary_experience ?
       default_video_bitrate_kbps[device_type] :
       default_non_primary_video_bitrate_kbps
-bitrate =
+video_bitrate_kbps =
       creative_video_bitrate_kbps ??
       creative_video_size_bytes * 8 / 1024 / creative_video_duration  ??
-      default_bitrate_kbps
+      default_bitrate
```

Comment:
why not kbps?

Reply:
The was actually another variable being used. Right fix was to fix the other variable to have `kbps` so now its logical